main.py

```diff
@@ -8,7 +8,30 @@
 from steps import STEPS
 from db import DB, DBs
 import typer
+from dotenv import load_dotenv
 
+load_dotenv()
+
+api_key = os.getenv("OPENAI_API_KEY") 
+# Check if OPENAI_API_KEY is defined
+if api_key == None:
+    # Raise an error if OPENAI_API_KEY is not defined
+    raise ValueError("OPENAI_API_KEY is not defined")
+else:
+    # Set the OPENAI_API_KEY to the api_key variable
+    openai.api_key = api_key
+
+# Get the OPENAI_API_BASE
+api_base = os.getenv("OPENAI_API_BASE")
+# Check if OPENAI_API_BASE is defined
+if api_base != None:   
+    # Set the OPENAI_API_BASE to the api_base variable
+    openai.api_base = api_base
+
+# Get the GPT model from the environment variable
+gpt_model = os.getenv("gpt_model")
+# Get the GPT temperature from the environment variable
+gpt_temperature = os.getenv("gpt_temperature")
```

Comment:
For these we could fall back into a default value:

```python
openai.api_base = os.getenv("OPENAI_API_BASE", "https://api.openai.com/v1")
gpt_model = os.getenv("gpt_model", "gpt-3.5-turbo")
gpt_temperature = os.getenv("gpt_temperature", 0.1)
```

Reply:
line 16 in https://github.com/AntonOsika/gpt-engineer/blob/12cb93fc7a6ea3d1865b6dd8a2f892c4105bebaf/scripts/rerun_edited_message_logs.py also has hard-coded reference to GPT-4...